README.md

```diff
@@ -261,82 +271,56 @@ Renders the widget to the DOM, and passes control back to your app through succe
+  tokens: Optional<Object>,
+  type: Optional<String>,
+  user: Optional<Object>,
+  stepUp: Optional<Function>,
+  session: Optional<Object>,
+  next: Optional<Function>,
+})
+```
 
-    // The user has started the unlock account flow, and is on the confirmation
-    // screen letting them know that an email is on the way.
-    if (res.status === 'UNLOCK_ACCOUNT_EMAIL_SENT') {
-      // Any followup action you want to take
-      return;
-    }
+- `status` *(string)* - Always present. One of: `FORGOT_PASSWORD_EMAIL_SENT`, `UNLOCK_ACCOUNT_EMAIL_SENT`, `ACTIVATION_EMAIL_SENT`, `REGISTRATION_COMPLETE`, or `SUCCESS`
 
-    // The user has successfully completed the authentication flow
-    if (res.status === 'SUCCESS') {
+- `username` *(optional\<string\>)* - Only present when `status` is one of `FORGOT_PASSWORD_EMAIL_SENT`, `UNLOCK_ACCOUNT_EMAIL_SENT`, `ACTIVATION_EMAIL_SENT`, or `REGISTRATION_COMPLETE`.
 
-      // Handle success when the widget is not configured for OIDC
+- `activationToken` *(optional\<object\>)* - Only present when `status` is `REGISTRATION_COMPLETE`.
 
-      if (res.type === 'SESSION_STEP_UP') {
-        // Session step up response
-        // If the widget is not configured for OIDC and the authentication type is SESSION_STEP_UP,
-        // the response will contain user metadata and a stepUp object with the url of the resource
-        // and a 'finish' function to navigate to that url
-        console.log(res.user);
-        console.log('Target resource url: ' + res.stepUp.url);
-        res.stepUp.finish();
-        return;
-      } else {
-        // If the widget is not configured for OIDC, the response will contain
-        // user metadata and a sessionToken that can be converted to an Okta
-        // session cookie:
-        console.log(res.user);
-        res.session.setCookieAndRedirect('https://acme.com/app');
-        return;
-      }
+- `tokens` *(optional\<object\>)* - Only present when widget is in an OIDC configuration, and `status` is `SUCCESS`. Depending on the widget `responseType` configuration, this will contain an `accessToken` only or both `accessToken` and `idToken`.
 
+- `type` *(optional\<string\>)* - Only present when widget is in a non-OIDC configuration and `status` is `SUCCESS`. One of `SESSION_STEP_UP`, or `SESSION_SSO`.
 
-      // OIDC response
+- `user` *(optional\<object\>)* - Only present when widget is in a non-OIDC configuration, `status` is `SUCCESS`, and `type` is `SESSION_STEP_UP`.
 
-      // If the widget is configured for OIDC with a single responseType, the
-      // response will be the token.
-      // i.e. authParams.responseType = 'id_token':
-      console.log(res.claims);
-      signIn.tokenManager.add('my_id_token', res);
+- `stepUp` *(optional\<function\>)* - Only present when widget is in a non-OIDC configuration, `status` is `SUCCESS`, and `type` is `SESSION_STEP_UP`. `res.stepUp.finish()` call redirect the user to the URL at `res.stepUp.url`.
```

Comment:
what is `res.stepUp.url`. is it related to any config arguments when init SignInWidget ?

Reply:
I think it's the URL that gets returned from authentication, so it wouldn't be related to the config, however I'm not very familiar with how Step-Up session auth works.